libs/ci/github/github.go

```diff
@@ -445,6 +445,13 @@ func ConvertGithubPullRequestEventToJobs(payload *github.PullRequestEvent, impac
 			return nil, false, fmt.Errorf("failed to find workflow config '%s' for project '%s'", project.Workflow, project.Name)
 		}
 
+		var skipMerge bool
```

Comment:
we are also missing the same logic in `ConvertIssueCommentEventToJobs` function so that it is always passed to the job struct correctly

Reply:
Added the logic in here